recipes-support/onboard/onboard_%.bbappend

```diff
@@ -0,0 +1,41 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+# file://0001-add-xfce-to-autostart-onlyshowin.patch
```

Comment:
Leftover comment?

Reply:
Sorry about that, I have removed this now.